config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   resources :agencies
   devise_for :users, controllers: { registrations: "registrations" }
-  root 'home#index'
+  root 'reports#index'
```

Comment:
since we no longer have a route to reach home#index, there's a build failure. should we just remove the home controller altogether?